### ISSUE TITLE
Modular laptops

### ISSUE THREAD

**Issue by anubhavkini** [system76/laptop-suggestions#145]
### Why/User Benefit/User Problem
Easy repairs, upgrades and customization.
### Description of the feature
There's a new company called [Framework](https://frame.work/) that's releasing a modular laptop. I think it's a great idea and maybe System76 would be interested.

**Comment by TeamLinux01** [comment]
Now that the framework is starting to ship, it might be possible to make compatible and interchangeable parts for S76 and framework machines, if S76 wants to join in that front.

I personally would enjoy a modular laptop that is repairable/upgradable with open firmware and EC.

**Comment by adeptg** [comment]
It looks like the Framework laptop [works well enough with Linux](https://www.phoronix.com/scan.php?page=article&item=framework-laptop&num=1). Having an OSS firmware and PopOS on the Framework would be nice.

**Comment by curiousercreative** [comment]
Just partner with Framework. They've already figured out hardware design and manufacturing.

**Comment by adeptg** [comment]
@curiousercreative   agree. But this way is so obvious that I think, they have already tried. And, as we don't see results yet, means that, for now, it doesn't possible for some reason.
Personally, I would buy a Framework laptop from System76 with a little premium for OS and firmware support
